src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 40 * days;
+  let discount = 0;
+
+  if (days >= 3) {
+    discount += 20;
+  }
+
+  if (days >= 7) {
+    discount += 30;
```

Comment:
The current logic applies both discounts cumulatively if the number of days is 7 or more. If the intention is to apply only the highest applicable discount, you should adjust the logic to apply a $20 discount for 3-6 days and a $50 discount for 7 or more days, rather than adding $30 to the $20 discount.